src/platform/mhv/secondary-nav/sass/mhv-sec-nav.scss

```diff
@@ -18,23 +15,26 @@ $nav-abbr-breakpoint: 415px;
   }
 
   a:hover {
-    color: var(--vads-color-primary);
+    color: var(--vads-color-primary-dark);
     background-color: rgba(255, 255, 255, 0.6);
     border-radius: 5px;
   }
 
-  i {
-    height: 17px;
-    padding-right: 0.7rem;
+  va-icon {
+    padding-right: 0.4rem;
```

Comment:
Given that the base font size is increasing, all the `rem` values will increase. So this might need to change in coordination with the DS changes… 

see also [this discussion about sizes](https://dsva.slack.com/archives/C04DRS3L9NV/p1715014885889939)